scripts/test_config.py

```
@@ -0,0 +1,299 @@
+            try:
+                content = test_file.read_text()
+                test_functions = [line for line in content.split('\n') 
+                                if line.strip().startswith('def test_')]
+                analysis["total_test_functions"] += len(test_functions)
+                
+                # Count markers
+                for line in content.split('\n'):
+                    line = line.strip()
+                    if line.startswith('@pytest.mark.'):
+                        marker = line.split('.')[2].split('(')[0]
+                        analysis["test_markers"][marker] = analysis["test_markers"].get(marker, 0) + 1
```

Ruff (0.11.9)

Line too long (102 > 88)

(E501)